### PR TITLE
SEO fixes for duplicate title tags and meta descriptions

### PR DESCRIPTION
**Pull request by tofuni**
This PR addresses duplicate title tags and meta descriptions which are part of the following issues:

- fixes https://github.com/pulumi/registry/issues/2285
- fixes https://github.com/pulumi/registry/issues/2286

### CODE REVIEW

**Comment by susanev** [comment]
thank you @tofuni!! this looks good to me, but want to run locally so gonna hold this for a bit until i have time to do that.

File: themes/default/content/blog/pulumi-challenge-september/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "Pulumi Challenge: One Quickstart to Rule them All"
 
 date: 2022-09-30T9:24:50-07:00
 
-meta_desc: Try Pulumi out for yourself by taking a Pulumi Challenge! Learn how it works, and get super secret swag in return.
+meta_desc: The Pulumi Challenge continues! Take this month’s challenge and win another unique piece of swag!
```
Comment (SaraDPH) [inline themes/default/content/blog/pulumi-challenge-september/index.md:6]:
```suggestion
meta_desc: Take the Pulumi Challenge - use a template to build a serverless application with AWS API Gateway and Lambda in less than 5 mins.
```

**Review by SaraDPH — CHANGES_REQUESTED** [review]
I changed the Pulumi Challenge meta description

**Review by SaraDPH — APPROVED** [review]

**Comment by tofuni** [comment]
Awesome - thank you @susanev and @SaraDPH!!

**Review by susanev — APPROVED** [review]

**Pull request merged**